Merian/main.yaml

```diff
@@ -0,0 +1,7 @@
+sources:
+  Wband:
```

Comment:
We usually try to find a unique name for the instrument. I am wondering if there's something more specific.

Reply:
so, the radar official name is FMCW 94 Ghz dual polarization radar, and it is from RPG. I think the best specification would be FMCW94_RPG. Also FMCW_94 could work. what do you prefer?

Reply:
I personally like FMCW94_RPG more, because it identifies the company and this is important for the radar..

Reply:
@d70-t ... changing the source name is enough or shall I also change the yaml file name? please let me know :)

Reply:
ok, changed the filename too 👍